code/game/objects/machinery/cic_maptable.dm

```diff
@@ -28,6 +24,8 @@
 		return
 	if(!user.client)
 		return
+	if (!map)
+		map = SSminimaps.fetch_minimap_object(targetted_zlevel, allowed_flags)
```

Comment:
Better would probably be to put this on lateinitialize, reason it gets a null map is because minimaps initializes AFTER atoms
better wuld be a queue like markers but this will do just as well for now

Reply:
LateInitialize doesn't work due to A.LateInitialize() getting called right after A.Initialize() inside the Atoms subsystem init itself.
And the queue markers.... i have no idea.